assets/oauth-openshift/oauth-server-config.yaml

```diff
@@ -26,9 +26,7 @@ oauthConfig:
     method: deny
     serviceAccountMethod: prompt
 {{ if .IdentityProviders }}  identityProviders:
-{{ .IdentityProviders | indent 2 }}
-{{- else }}  identityProviders: []
-{{- end -}}
+{{ trimTrailingSpace .IdentityProviders | indent 2 }}{{- else }}  identityProviders: []{{- end }}
```

Comment:
In the case where `.IdentityProviders` was empty, the trailing `-` in the final `{{ end }}` was causing the next line (starting with `loginURL` to get appended to identityProviders, resulting in invalid YAML. With this change, the `.IdentityProviders` is trimmed of a final newline, and the newline is provided by the template. Tested with/without a value for .IdentityProviders and worked as expected with both.